config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
 
-  root 'pages#index'
+  # root 'pages#index'
```

Comment:
@shivamsupr Can you map this to `/login` so that we can keep the authentication functionality accessible as well?

Reply:
@BilalBudhani done, '/login' route can be accessed now.